server/subscription.go

```diff
@@ -45,10 +45,10 @@ func (s *Server) registerSubscriptionRoutes(g *echo.Group) {
 // Return subscription with free plan if no license found.
 func (s *Server) loadSubscription() *enterpriseAPI.Subscription {
 	subscription := &enterpriseAPI.Subscription{
-		Plan: api.FREE,
+		Plan: api.ENTERPRISE,
 		// -1 means not expire, just for free plan
 		ExpiresTs:     -1,
-		InstanceCount: 5,
+		InstanceCount: 9999,
```

Comment:
Maybe keep default to FREE with a maximum of 20 or 30 instance, and change feature matrix to all true for free? Otherwise, users may get impression that they got ENTERPRISE tier already.